build/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../tsconfig-base.json",
+  "compilerOptions": {
+    "outDir": "../.tmp/tsbuildinfo/build",
+  },
+  "references": [
+    {"path": "../"},
+    {"path": "../treemap/"},
+  ],
+  "include": [
+    "**/*.js",
+    "../types/*.d.ts",
```

Comment:
Actually, this is kind of messy. This and `{"path": "../"}` means the dependency tree is essentially everything, there's not really anything gained by separating it, and there's the risk of future incompatibilities because it's pulling in root global types _and_ treemap code, and some of our report global types clash with the core global types (which was the original impetus for splitting these up).

We could do it the right way (make a `build/types/` directory, move the types needed in there, have explicit imports for needed types (e.g. [`viewer.d.ts`](https://github.com/GoogleChrome/lighthouse/blob/d9929e0a0f6a14059f00a3386873f34c5a1546d1/viewer/types/viewer.d.ts) style), but it's probably not worth it.

> `{"path": "./treemap/"},` in the root `tsconfig.json`

is probably the best way to go